Uplift/Supporting/Constants.swift

```diff
@@ -1,12 +1,20 @@
 //
-//  File.swift
+//  Constants.swift
 //  Uplift
 //
 //  Created by Joseph Fulgieri on 8/31/18.
 //  Copyright © 2018 Uplift. All rights reserved.
 //
 
 import Foundation
```

Comment:
delete since UIKit imports Foundation inside

Reply:
you're right, good catch